src/Integration.Vsix.UnitTests/CFamily/ProtocolTest.cs

```diff
@@ -72,25 +72,14 @@ public void Read_Empty_Response()
         }
 
         [TestMethod]
-        public void Read_RequestHasFileName_ReturnedMessageHasNoFileName_MessageIgnored()
+        public void Read_RequestHasFileName_ReturnedMessageHasNoFileName_MessageIsNotIgnored()
```

Comment:
Changed of behaviour in the protocol reader - it should pass all messages to the message handler for processing.